### DISCUSSION TITLE
Consider unhiding --color-moved by default

### DISCUSSION THREAD

**Discussion by matklad** [magit/magit#4797] [Ideas]
`--color-moved` is an extremely useful feature of git, and one which is easy to miss (i've been programming for quite some time,and learned about it just this year). 

Today, magit hides it by default (https://github.com/magit/magit/issues/4254#issuecomment-731874472). I think it would be better if it was shown:

* this is generally a useful thing to enable for many refactors
* there's _tremendous_ value in tehaching people that this thing exists (speaking as someone who learned about `--force-with-lease` thanks to magit)

**Comment by tarsius** [comment]
I have once more decided against showing it by default.
